components/cdc/src/endpoint.rs

```diff
@@ -833,7 +831,7 @@ impl<T: 'static + CdcHandle<E>, E: KvEngine, S: StoreRegionMeta> Endpoint<T, E,
         let release_scan_task_counter = tikv_util::DeferContext::new(move || {
             scan_task_counter.fetch_sub(1, Ordering::Relaxed);
         });
-        if scan_task_count + 1 > self.config.incremental_scan_concurrency_limit as isize {
+        if scan_task_count == self.config.incremental_scan_concurrency_limit as isize {
```

Comment:
Please explain why change this condition here.

Reply:
I think they are the same condition
if a + 1 > b, which always indicates that a >= b.